File: Assets/ApagarDiretorio.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using System.IO;

public class ApagarDiretorio : MonoBehaviour {
	public string _local;
	
	void Start () {
		_local = Application.dataPath;
		print (_local);
		if (Directory.Exists (_local + "/teste(c#)")) {
			Directory.Delete(_local + "/teste(c#)");
		} else {
			print("Esse diretorio nao existe.");
		}
	}

	void Update () {}
}

File: Assets/ApagarDiretorio.js
#pragma strict
import System.IO;

public var _local : String;

function Start () {
	_local = Application.dataPath;
	print(_local);
	if(Directory.Exists(_local+"/teste(javascript)")){
		Directory.Delete(_local + "/teste(javascript)");		
	}else{
		print("Esse diretorio nao existe.");
	}
}

function Update () {}
